File: backend-python/app/modules/system/dto/dict_request.py
from pydantic import BaseModel,Field
class DictTypeSaveRequest(BaseModel):
 dict_name:str=Field(alias="dictName");dict_code:str=Field(alias="dictCode");status:int|None=None;remark:str|None=None
class DictItemSaveRequest(BaseModel):
 dict_type_id:int=Field(alias="dictTypeId");label:str;value:str;is_default:int|None=Field(None,alias="isDefault");sort:int|None=None;status:int|None=None

File: backend-rust/src/modules/system/dto/dict_request.rs
use serde::Deserialize;

#[derive(Debug, Clone, Deserialize)]
#[serde(rename_all = "camelCase")]
pub struct DictTypeSaveRequest {
    pub dict_name: String,
    pub dict_code: String,
    pub status: Option<i32>,
    pub builtin: Option<i32>,
    pub remark: Option<String>,
}

#[derive(Debug, Clone, Deserialize)]
#[serde(rename_all = "camelCase")]
pub struct DictItemSaveRequest {
    pub dict_type_id: i64,
    pub label: String,
    pub value: String,
    pub is_default: Option<i32>,
    pub status: Option<i32>,
    pub sort: Option<i32>,
    pub builtin: Option<i32>,
}
